### PR TITLE
Branch storage tests

### LINKED ISSUE #162: Tests for Storage related stuff

### PR DESCRIPTION
**Pull request by lrj689**
Tests added, storage read catches overlapping entries

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAdaptedEntry.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import seedu.address.commons.core.Messages;
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/storage/JsonAdaptedEntry.java]: wanna do `import static seedu.address.commons.core.Messages.MESSAGE_INVALID_...` instead?

File: src/test/java/seedu/address/logic/commands/DeleteEntryCommandTest.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 public class DeleteEntryCommandTest {
 
-    private Model model = new ModelManager(getTypicalEntriesList(), new UserPrefs());
+    private Model model = new ModelManager(getTypicalTeachingAssistant(), new UserPrefs());
```
Comment (nicholastanvis) [inline src/test/java/seedu/address/logic/commands/DeleteEntryCommandTest.java:23]: lovely

File: src/test/java/seedu/address/testutil/TypicalTeachingAssistant.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            ab.addEntry(entry);
+        }
+        return ab;
+    }
+
+    public static List<Entry> getTypicalEntries() {
+        return new ArrayList<>(Arrays.asList(CONSULTATION, CLASS_MEETING, EXTRA_CLASS, SHORT_QUIZ, DO_STUFF));
+    }
+
+    public static List<Person> getTypicalPersons() {
```
Comment (nicholastanvis) [inline src/test/java/seedu/address/testutil/TypicalTeachingAssistant.java:102]: in this pr, i don't see this method being used anywhere. can i assume that removing this breaks the tests?

**Review by nicholastanvis — APPROVED** [review]

File: src/test/java/seedu/address/testutil/TypicalTeachingAssistant.java (hunk 3 above)
Comment (lrj689) [reply src/test/java/seedu/address/testutil/TypicalTeachingAssistant.java:102]: Used by getTypicalTeachingAssistant method

**Review by tsh22 — APPROVED** [review]

**Review by goatygoatygoat — APPROVED** [review]

**Pull request merged**